testworld/nft_test.go

```diff
@@ -130,10 +132,14 @@ func paymentObligationMint(t *testing.T, documentType string, grantNFTAccess, to
 	if status != "success" {
 		t.Error(message)
 	}
-
-	assert.Nil(t, err, "mintNFT should be successful")
+	assert.NoError(t, err, "mintNFT should be successful")
 	assert.True(t, len(response.Value("token_id").String().Raw()) > 0, "successful tokenId should have length 77")
 
+	tokenID, err := nft.TokenIDFromString(response.Value("token_id").String().Raw())
+	assert.NoError(t, err, "token ID should be correct")
+	owner, err := alice.host.tokenRegistry.OwnerOf(registry, tokenID.BigInt().Bytes())
+	assert.NoError(t, err)
+	assert.Equal(t, strings.ToLower("0x44a0579754d6c94e7bb2c26bfa7394311cc50ccb"), strings.ToLower(owner.Hex()))
```

Comment:
I think you meant to pass here the deposit address of the test context.

Reply:
we always deposit to our address to make to debug easily

Reply:
even against local envs?

Reply:
yes, its the same hardcoded address for all